content/articles/packet-filtering-using-acls/index.md

```diff
@@ -0,0 +1,173 @@
+
+- Extended ACLs- packet filtering is done based on source IP, destination IP, protocol types, and port numbers.
+They range from 100-199, implemented closest to the source and a selected number of services are blocked as specified by the network administrator.
+
+Command **access-list?** is used to show the available ACLs that can be configured on the router interface.
+
+
+```bash
+R1(config)# access-list ?
+ <1-99> IP standard access list
+ <100-199> IP extended access list
+ <1100-1199> Extended 48-bit MAC address access list
+ <1300-1999> IP standard access list (expanded range)
+ <200-299> Protocol type-code access list
+ <2000-2699> IP extended access list (expanded range)
+ <700-799> 48-bit MAC address access list
+ rate-limit Simple rate-limit specific access list
+ template Enable IP template acls
+Router(config)# access-list
+```
+
+#### Access Control Lists Tasks.
+Routers use Access Control Lists to perform various tasks:
+
+- Increase network performance by limiting the network traffic
+
+- Prioritization of certain network traffic
+
+- A level of network security access
+
+- Traffic flow control
+
+
+### Access Control List Configuration.
+We will learn how to create and configure standard IPv4 ACLs and how they filter the traffic.
+To create a named standard ACL, we use the following global configuration command;
+`Router(config)# ip access-list standard access-list-name`
+For instance, we have created an access list called PERMIT-ACCESS of the standard type.
+```bash
+R1(config)# ip access-list standard PERMIT-ACCESS
+R1(config-std-nacl)# remark ACE permits host 192.168.10.10
+R1(config-std-nacl)# permit host 192.168.10.10
+R1(config-std-nacl)#
+
+```
+For the numbered one we use command: `Router(config)# access-list access-list-number {deny | permit | remark text} source [source-wildcard] [log]`
+
+For instance, we have created ACL 10 which permits a particular host to the internet.
+```bash
+R1(config)# access-list 10 remark ACE permits ONLY host 192.168.10.10 to the internet
+R1(config)# access-list 10 permit host 192.168.10.10
+R1(config)# do show access-lists
+Standard IP access list 10
+    10 permit 192.168.10.10
+R1(config)#
+```
+Lets consider the  Local Area Network below:
+![CDP-LLDP Network Configuration](/engineering-education/packet-filtering-using-acls/acl.jpg)
+
+We will create one numbered ACL, ACL 20 that denies host 192.168.10.10 but permits all other hosts on network 192.168.10.0/24. Start by configuring the ACL 20 ACE that denies the 192.168.10.10 host using the `host` keyword, then create the ACL that permits all other hosts.
+```bash
+R1(config)#access-list 20 deny host 192.168.10.10
+R1(config)#access-list 20 permit 192.168.10.0 0.0.0.255
+```
+Since ACL 20 only apply to traffic from LAN 1, the ACL would be applied to the incoming traffic to the G0/0/0 R1 interface.
+Enter interface g0/0/0 mode, apply ACL 20 inbound and return to global configuration mode.
```

Comment:
```suggestion
Since ACL 20 only apply to traffic from LAN 1, the ACL can be applied to the incoming traffic to the G0/0/0 R1's interface.
Enter interface g0/0/0 mode, apply ACL 20 inbound and return to global configuration mode.
```